08_education/college_metrics_county.qmd

```diff
@@ -0,0 +1,1171 @@
+  labs(x = "independent_living_difficulty",
+       y = NULL)+
+  scale_x_discrete(labels = label_wrap(10))
+
+```
+
+* IPUMS documented values for ambulatory_difficulty: 
+    + 0 N/A
+    + 1 No ambulatory difficulty
+    + 2 Has ambulatory difficulty 
+  
+Plot the options for ambulatory_difficulty below.
+```{r}
+
+acs_clean %>% 
+  count(ambulatory_difficulty) %>% 
+  ggplot(mapping = aes(x = ambulatory_difficulty, y = n)) +
+  geom_col() +
+  theme_minimal() +
+  ggtitle("Ambulatory difficulty, All Records") +
+ geom_text(mapping = aes(label = n), vjust = -1) +    
+  scale_y_continuous(expand = expansion(mult = c(0, 0.1))) +
+  labs(x = "ambulatory_difficulty",
+       y = NULL)+
+  scale_x_discrete(labels = label_wrap(10))
+
+```
+  
+* IPUMS documented values for cognitive_difficulty: 
+    + 0 N/A
+    + 1 No cognitive difficulty
+    + 2 Has cognitive difficulty 
+  
+Plot the options for cognitive_difficulty below.
+```{r}
+acs_clean %>% 
+  count(cognitive_difficulty) %>% 
+  ggplot(mapping = aes(x = cognitive_difficulty, y = n)) +
+  geom_col() +
+  theme_minimal() +
+  ggtitle("Cognitive difficulty, All Records") +
+ geom_text(mapping = aes(label = n), vjust = -1) +    
+  scale_y_continuous(expand = expansion(mult = c(0, 0.1))) +
+  labs(x = "cognitive_difficulty",
+       y = NULL)+
+  scale_x_discrete(labels = label_wrap(10))
+
+```
+
+Create the subgroup for disability. 
+```{r}
+acs_clean <- acs_clean %>% 
+  mutate(subgroup_disability = case_when(cognitive_difficulty == "Has cognitive difficulty" |
+                                           ambulatory_difficulty == "Has ambulatory difficulty" |
+                                           independent_living_difficulty == "Has independent living difficulty" |
+                                            vision_hearing_difficulty == "Has vision or hearing difficulty" |
+                                           self_care_difficulty == "Yes" ~ "with disability",
+                                         TRUE ~ "without disability"))
+  
+```
+
+Check the variable breakout. 
+```{r}
+acs_clean %>% 
+  count(subgroup_disability) %>% 
+  ggplot(mapping = aes(x = subgroup_disability, y = n)) +
+  geom_col() +
+  theme_minimal() +
+  ggtitle("Subgroup Disability, All Records") +
+ geom_text(mapping = aes(label = n), vjust = -1) +    
+  scale_y_continuous(expand = expansion(mult = c(0, 0.1))) +
+  labs(x = "Disability",
+       y = NULL)+
+  scale_x_discrete(labels = label_wrap(10))
+```
+
+## Crosswalk
+
+Read in the PUMA to county crosswalk file. This file is created by the program generate_puma_county_crosswalks.rmd in the geographic-crosswalks folder. 
+```{r}
+puma_county_crosswalk <- read_csv(here::here("geographic-crosswalks", "data", "crosswalk_puma_to_county.csv"))
+```
+
+## Merge Crosswalk 
+
+Create a variable in the ACS data indicating whether the survey is pre or post-2022. This will be used to join on the correct crosswalk information.
```

Comment:
Just a note that, with the change in datasets to 5-year estimates, nothing has a `crosswalk_period` of "2022" at this point, right?